routes/events.js

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import {
+  getEventByDepartment,
+  getEventByName,
+  deleteEvent,
+  getAllEvents,
+} from "../controllers/eventController";
+
+const router = express.Router();
+router.get(":/department_event", getEventByDepartment);
+router.get(":/name_event", getEventByName);
+router.get(":/events", getAllEvents);
```

Comment:
Put the colons after the route names, and make them :id so they actually get parsed